src/Stuart/Client.php

```diff
@@ -89,17 +110,19 @@ public function getEta($job)
         $body = JobToJson::convert($job);
 
         $apiResponse = $this->httpClient->performPost($body, '/v2/jobs/eta');
+
         return json_decode($apiResponse->getBody());
     }
 
     private function validateAddress($address, $type)
     {
         $query = array(
             'address' => $address,
-            'type' => $type
+            'type' => $type,
         );
 
         $apiResponse = $this->httpClient->performGet('/v2/addresses/validate', $query);
+
```

Comment:
As this PR doesn't touch the address validation method, we should discard this line.